File: reactjs/src/login/gql/mutations.js
import { gql } from "@apollo/client";

export const LOGIN_USER = gql`
  mutation LoginUser($username: String!, $password: String!) @api(name: app){
    loginUser(username: $username, password: $password) {
      user {
        id
        username
        email
        isStaff
      }
    }
  }
`;


export const LOGOUT_USER = gql`
  mutation LogoutUser @api(name: app){
    logoutUser{
      success
    }
  }
`


export const CREATE_USER = gql`
  mutation CreateUser($username: String!, $password: String!, $email: String!) @api(name: app){
    createUser(username: $username, password: $password, email:$email) {
      user {
        id
        username
        email
      }
    }
  }
`;

File: shop/auth/gql/mutations.py

import graphene

import auth.gql.mutations_logic as mutations_logic


class RegistrationMutations(graphene.ObjectType):
    create_user = mutations_logic.CreateUser.Field()
    login_user = mutations_logic.LoginUser.Field()
    logout_user = mutations_logic.LogoutUser.Field()
